source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts

```diff
@@ -7,18 +7,18 @@ export * from "./adaptivecards-designer-standalone";
 export * from "./containers/index";
 
 export const defaultMicrosoftHosts: containers.HostContainer[] = [
```

Comment:
nit: Personally, I would make the two added-on to use as less as space as possible.
1. remove the Seperator in between
2. `Container Size` and `Theme` is enough. If you think it's not obvious, you can add tooltip to provide more information

![image](https://user-images.githubusercontent.com/6290692/185208463-dbd874a0-c314-40a2-8534-b96adee7038d.png)